fouleggs/src/de/up/hpi/informationsystems/fouleggs/dactors/AdminSession.scala

```diff
@@ -0,0 +1,176 @@
+package de.up.hpi.informationsystems.fouleggs.dactors
+
+import akka.actor.Actor.Receive
+import akka.actor.{Actor, ActorLogging, ActorRef, ActorSystem, Props}
+import de.up.hpi.informationsystems.adbms.Dactor
+import de.up.hpi.informationsystems.adbms.protocols.DefaultMessagingProtocol
+import de.up.hpi.informationsystems.adbms.record.Record
+import de.up.hpi.informationsystems.adbms.record.ColumnCellMapping._
+import de.up.hpi.informationsystems.adbms.relation.Relation
+
+object AdminSession {
+
+  final case class Up()
+
+  object AddCastToFilm {
+    final case class Request(personId: Int, filmId: Int, roleName: String)
+    final case class Success(personId: Int, filmId: Int, roleName: String)
+    final case class Failure(e: Throwable)
+  }
+
+  def props: Props = Props[AdminSession]
+}
+
+/**
+  * Provides top level functionalities
+  */
+class AdminSession extends Actor with ActorLogging {
+  override def receive: Receive = commonBehaviour
+
+  def commonBehaviour: Receive = {
+    case AdminSession.Up => sender() ! akka.actor.Status.Success
```

Comment:
I don't get the point of this one. If it's for a health-check, then you could use `Identify` and `ActorIdentity` or just rely on the parent-child-relationship and listen in the `SystemInitializer` for a `Terminated` message.

Nothing, you have to change now.